.github/templates/main.yml

```diff
@@ -103,6 +103,16 @@ steps:
 #@ end
 #@ end
 
+#@ def publishTestsResults(files, test_title):
+  - name: Publish Unit Test Results
```

Comment:
This can return a simple object rather than array element. That way we can avoid the `template.replace` when consuming the function.